delfin/drivers/huawei/oceanstor/alert_handler.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from oslo_log import log
+
+from delfin import exception
+from delfin.i18n import _
+
+LOG = log.getLogger(__name__)
+
+
+class AlertHandler(object):
+    """Alert handling functions for huawei oceanstor driver"""
+    default_me_category = 'storage-subsystem'
+
+    def __init__(self):
+        pass
+
+    """
+    Alert model contains below fields
+    category : Type of the reported notification
+    occur_time : Time of occurrence of alert. When trap does not contain it,
+                 it will be filled with receive time
+    match_key : This info uniquely identifies the fault point. Several infos
+                such as source system id, location, alarm id together can be
+                used to construct this
+    me_dn : Unique id at resource module (management system) side. me stands
```

Comment:
Please use a normal name for those parameters.

Reply:
I would say , name changes can be taken later. Let us go with these names in Faroe release

Reply:
Raised an issue for this improvement
https://github.com/sodafoundation/delfin/issues/239